tests/framework/report.py

```diff
@@ -199,14 +201,29 @@ def parse_data(test):
                         item_value not in crt_doc_item.one_of:
                     raise ValueError(
                         f"{crt_item} must be one of \
-{crt_doc_item.one_of}, not {item_value}")
+                        {crt_doc_item.one_of}, not {item_value}")
```

Comment:
The exception here will look like:

"Blah must be one of                                   blah, not blah".

The previous way this was formatted is not ideal, indeed, but we can also do:
```python

raise ValueError(
    f"{crt_item} must be one of"\
    "{crt_doc_item.one_of}, not {item_value}")
```

Reply:
done